coalib/parsing/LineParser.py

```diff
@@ -56,6 +57,20 @@ def parse(self, line):
             line,
             self.comment_separators)
         comment = unescape(comment)
+
+    # MY CODE
```

Comment:
ok this seems to be debug code, please add a `wip: ` prefix to your PR title while you're working on it